packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -106,25 +115,32 @@
       },
       "properties": {
         "entrypoint": {
+          "description": "The entrypoint of the first flow in the session.",
           "$ref": "#/properties/user_properties/definitions/entrypointPattern"
         },
         "entrypoint_experiment": {
+          "description": "Identifier for the experiment the user is part of at the entrypoint (if any).",
           "$ref": "#/properties/user_properties/definitions/entrypointPattern"
         },
         "entrypoint_variation": {
+          "description": "Identifier for the experiment variation the user is part of at the entrypoint (if any).",
```

Comment:
These definitions are a little circular, and at least for entrypoint, that does not seem like an accurate description. It looks like they're defined here: https://mozilla.github.io/ecosystem-platform/relying-parties/reference/metrics-for-relying-parties#metrics-related-query-parameters. We could pilfer from that?

Reply:
I had pilfered from `fxa-auth-server/docs/metrics-events.md`, but I'll pilfer from Ecosystem Platform. Thanks!